schedule/publiccloud/mau-extratests-publiccloud.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: mau-extratests-publiccloud
+description: |
+  Run mau-extratests on public cloud instances
+schedule:
+  - boot/boot_to_desktop
+  - publiccloud/download_repos
+  - publiccloud/ssh_interactive_init
+  - publiccloud/register_system
+  - publiccloud/transfer_repos
+  - publiccloud/patch_and_reboot
+  - publiccloud/ssh_interactive_start
+  - publiccloud/instance_overview
+  - console/system_prepare
+  - console/prepare_test_data
+  - console/consoletest_setup
+  - console/check_os_release
+  - console/cleanup_qam_testrepos
+  - console/openvswitch
+  - console/shar
+  - console/sshd
+  - console/update_alternatives
+  - console/rpm
+  - console/slp
+  - console/pkcon
+  - console/libvorbis
+  - console/command_not_found
+  - console/openssl_alpn
+  - console/autoyast_removed
+  - console/cron
+  - console/syslog
+  - console/mta
+  - console/check_default_network_manager
+  - console/git
+  - console/cups
+  - console/java
+  - console/sqlite3
+  - console/gdb
+  - console/sysctl
+  - console/sysstat
+  - console/wget_ipv6
+  - console/ca_certificates_mozilla
+  - console/unzip
+  - console/gpg
+  - console/rsync
+  - console/clamav
+  - console/shells
+  - console/sudo
+  - console/dstat
+  - x11/evolution/evolution_prepare_servers
+  - console/mutt
+  - console/supportutils
+  - console/journalctl
+  - console/quota
+  - console/rpcbind
+  - sysauth/sssd
+  - console/timezone
+  - console/procps
+  - console/lshw
+  - console/kmod
+  - console/suse_module_tools
+  - console/zziplib
+  - console/firewalld
+  - console/aaa_base
+  - console/osinfo_db
+  - console/libgcrypt
+  - console/gd
+  - console/valgrind
+  - publiccloud/ssh_interactive_end
+  # - TAP/__sqlite3_run-tests_t
```

Comment:
this line does not have any meaning for me , can you elaborate why we need it ?

Reply:
i have this PR still as draft. and actually i left it for reviewer because i do not understand why (i did not actual search the why :=) ) it appears and if actual is needed.

Reply:
yeah sure totally fine by me . just saying that before merge we need to do one of the following : 
1. find the answer why it should be here 
2. remove it 
:)

Reply:
Yiannis pointed me to [this](https://openqa.suse.de/tests/5093263) run today. The `TAP/__sqlite3_run-tests_t` is the result output produced by `console/sqlite3` test module so we should in remove this line.

Reply:
@asmorodskyi i have removed it. it seems it is the results of tests/console/sqlite3.pm